bin/uglifyjs

```diff
@@ -110,7 +110,7 @@ You need to pass an argument to this option to specify the name that your module
 
     .boolean("expr")
     .boolean("source-map-include-sources")
-    .boolean("screw-ie8")
+    .boolean("support-old-ie")
```

Comment:
We can enable screw-ie8 by default but can't drop support for the CLI `--screw-ie8` flag used in the wild.